src/log_surgeon/finite_automata/RegexDFA.hpp

```diff
@@ -23,11 +23,15 @@ class RegexDFAState {
 public:
     using Tree = UnicodeIntervalTree<RegexDFAState<stateType>*>;
 
-    auto add_tag(int const& rule_name_id) -> void { m_tags.push_back(rule_name_id); }
+    auto add_matching_variable_id(int const& variable_id) -> void {
```

Comment:
```suggestion
    auto add_matching_variable_id(int variable_id) -> void {
```
Same, unless we have a reason to enforce using the const reference

Reply:
Changed to const, same reason as above.